Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG BUILD_FROM
+FROM ${BUILD_FROM}
+
+WORKDIR /usr/src
+
+## Setup Home Assistant
+COPY . homeassistant/
+RUN pip3 install --no-cache-dir --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5223963)